src/Bicep.Core/TypeSystem/DeployTimeConstantVariableVisitor.cs

```diff
@@ -13,25 +13,27 @@ namespace Bicep.Core.TypeSystem
     public sealed class DeployTimeConstantVariableVisitor : SyntaxVisitor
     {
         private readonly SemanticModel model;
-        public Stack<string> visitedStack;
```

Comment:
>visitedStack [](start = 29, length = 12)

Changed it because fields shouldn't have been directly exposed (should be encapsulated in properties even when exposing a setter is intended).